File: src/main/java/com/baulsupp/oksocial/services/foursquare/FourSquareUtil.java
package com.baulsupp.oksocial.services.foursquare;

import com.google.common.collect.Sets;
import java.util.Collections;
import java.util.Set;
import okhttp3.Request;

public class FourSquareUtil {
  private FourSquareUtil() {
  }

  public static final Set<String> API_HOSTS =
      Collections.unmodifiableSet(Sets.newHashSet(
          "api.foursquare.com")
      );

  public static Request apiRequest(String s, Request.Builder requestBuilder) {
    return requestBuilder.url("https://api.foursquare.com" + s).build();
  }
}

File: src/main/kotlin/com/baulsupp/oksocial/services/foursquare/FourSquareUtil.kt
package com.baulsupp.oksocial.services.foursquare

import okhttp3.Request

object FourSquareUtil {

  val API_HOSTS = setOf((
      "api.foursquare.com")
  )

  fun apiRequest(s: String, requestBuilder: Request.Builder): Request {
    return requestBuilder.url("https://api.foursquare.com" + s).build()
  }
}
